### ISSUE TITLE
Partials are not usable in _head.mustache nor _foot.mustache

### ISSUE THREAD

**Issue by illepic** [pattern-lab/patternlab-php#227]
Partials do not render when called from head or foot. For instance, in `_01-foot.mustache` we can try :

```
  {{> atoms-inline-data }}

  <script src="{{ base_path }}js/script.js"></script>

  <!--DO NOT REMOVE-->
  {% pattern-lab-foot %}

  </body>
</html>
```

But nothing is printed out of the `atoms/inline-data.mustache` file.

Being able to render partials in foot/head is useful for things like global inline templates that js needs to grab regardless of the pattern being currently viewed. Rendering partials in foot_er_.mustache still works great, but means we need to be viewing a full page pattern in order to get access to this "global" information.

**Comment by mimshwright** [comment]
+1 for being able to do this in some way. It would be nice to be able to have the global navigation as its own organism but also have it injected into the page template. Instead I need to add it to every template manually.

**Comment by dmolsen** [comment]
@mimshwright -

This is a larger issue than just "make partials usable in the header and footer." Currently there is one header and footer for *everything*. So the same header and footer goes on the "view all" views and every single pattern. Not just "pages" and "templates." So including a partial in the header for "pages" will also add that same partial everywhere else.

I tested switching the header and footer to being parsed by the pattern loader and the perf wasn't affected overly. That said, after some thought I realized the issue in the preceding paragraph sort of kills the idea. A better answer is probably to use "blocks" which is the mustache version of layouts. [More info](https://github.com/bobthecow/mustache.php/wiki/BLOCKS-pragma).

If you like blocks let me know and I'll look more closely at it.
